android/res/layout/view_vpn_status_detail.xml

```diff
@@ -48,49 +48,130 @@
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_marginTop="30dp" />
+
+
+        <ImageView
+            android:id="@+id/imageView"
+            style="@style/divider_line"
+            android:layout_width="match_parent"
+            android:layout_height="2dp"
+            android:layout_marginBottom="22dp"
+            android:layout_marginTop="22dp"
+            android:adjustViewBounds="false" />
+
+
+        <!-- VPN client's feature list -->
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginBottom="15dp"
+            android:layout_marginTop="5dp"
+            android:gravity="center_horizontal"
+            android:orientation="horizontal">
+
+            <TextView
+                android:id="@+id/VPN_what_do_you_get"
+                style="@style/VPNTitleSecond"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:paddingRight="5dp"
+                android:text="@string/VPN_what_you_get_with" />
+
+            <TextView
+                android:id="@+id/VPN_client"
+                style="@style/VPNTitleSecond.Color"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/VPN_client" />
+        </LinearLayout>
+
+        <TextView
+            android:id="@+id/VPNFeatures"
+            style="@style/VPNText.Regular"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center_horizontal"
+            android:text="@string/VPN_feature_list_html" />
 
         <!-- Bullet - VPN not required for FrostWire but highly recommended -->
-        <com.frostwire.android.gui.views.HTMLTextView
+        <TextView
+            android:id="@+id/vpn_bullet"
+            style="@style/VPNText"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:layout_marginTop="20dp"
             android:fontFamily="sans-serif-normal"
+            android:gravity="center_horizontal"
             android:padding="0dp"
-            android:text="@string/you_dont_need_a_vpn_to_use_frostwire_bullet_html"
             android:textColor="@color/app_text_secondary"
             android:textSize="@dimen/text_size_medium" />
 
+        <!-- ExpressVPN graphic -->
+        <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_marginBottom="10dp"
+            android:layout_marginTop="15dp"
+            android:src="@drawable/express_vpn" />
+
+        <!-- GetVPN bottom button -->
+        <Button
+            android:id="@+id/button_VPN"
```

Comment:
I'm anal about naming because otherwise it's really easy to have a mess and confuse ids and have null pointers.
Let's stick to our conventions they've worked out great over the years.